router.js

```diff
@@ -1,228 +1,278 @@
-'use strict';
+
 
 /**
  * A simple rooter to add routes to your express app and expose unifile api
  * https://github.com/silexlabs/unifile/
  * license: GPL v2
  */
-// node modules
+// Node modules
 const Path = require('path');
 const PassThrough = require('stream').PassThrough;
 const serveStatic = require('serve-static');
 const Unifile = require('unifile');
 
-// file upload
-const multer  = require('multer');
-const storage = multer.memoryStorage()
-const upload = multer({ storage: storage })
+// File upload
+const multer = require('multer');
```

Comment:
This should go in a PR related to the linter...

Reply:
You're right, my editor auto-linted. I'll make a PR to clean and lint the server side.